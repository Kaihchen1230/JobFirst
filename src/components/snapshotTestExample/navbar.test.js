import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import NavBar from "../navBar"

//a snapshot test example
describe("snapshot test",()=>{
  test('should render Header correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<NavBar />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
