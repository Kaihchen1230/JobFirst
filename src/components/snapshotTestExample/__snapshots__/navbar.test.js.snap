// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test should render Header correctly 1`] = `
<Menu
  className=""
  focusable={false}
  mode="horizontal"
  theme="dark"
>
  <MenuItem>
    <Icon
      theme="twoTone"
      type="home"
    />
    Home
    <ForwardRef
      to="/app/main"
    />
  </MenuItem>
  <MenuItem>
    <Icon
      theme="twoTone"
      type="profile"
    />
    View Job
    <ForwardRef
      to="/app/job-list"
    />
  </MenuItem>
  <MenuItem>
    <Icon
      theme="twoTone"
      type="mail"
    />
    Contact us
  </MenuItem>
  <MenuItem>
    <Icon
      theme="twoTone"
      type="mail"
    />
    Register
  </MenuItem>
  <MenuItem>
    <Icon
      theme="twoTone"
      type="profile"
    />
    Post a New Job
    <ForwardRef
      to="/app/postJob"
    />
  </MenuItem>
  <MenuItem>
    <ForwardRef
      to="/app/user-profile"
    >
      <span>
        <Avatar
          icon="user"
          shape="square"
          size="large"
        />
        <span>
           Log in 
        </span>
      </span>
    </ForwardRef>
  </MenuItem>
</Menu>
`;
